components/apimgt/org.wso2.carbon.apimgt.rest.api.dcr/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -12,7 +12,7 @@
 	  </property> 
     </bean> 
     <bean class="org.springframework.beans.factory.config.PreferencesPlaceholderConfigurer"/>
-    <jaxrs:server id="RegistrationService" address="/register">
+    <jaxrs:server id="RegistrationService" address="/">
```

Comment:
we had to remove this? any idea on the impact of this?

Reply:
yes. If we use /register base path becomes `/client-registration/v0.17/register` and hence resource become invalid. Since /register is a resource, remove it from here and defined it in the RegistrationServiceImpl.java